### ISSUE TITLE
Research control

### ISSUE THREAD

**Issue by Siliconlad** [automat-ed/autonomous#6]
For Demo 2, we need to be able to show that we can control the robot in such a way that it stays within the detected walkways in simulation. Before we can do this, we need to conduct some research into how to do this. In addition, we need to compile a list of required sensors to pass to the hardware team so that they can design the robot.

# Task
To conduct research into different methods for detecting controlling the robot the simulation. All research should be documented in the [Google Drive folder](https://drive.google.com/drive/folders/1m9Mdte_oveNHa0qymPkkvpYUwnPCyi6p?usp=sharing). In addition, a list of required sensors should be compile along with their position for optimal use.

**Comment by Siliconlad** [comment]
Superseded by #16
